### DISCUSSION TITLE
Creating an Editable Table Chart

### DISCUSSION THREAD

**Discussion by BigBatz** [apache/superset#19137] [General]
A customer of mine wants to have an editable column in the chat type "Table" to insert comments for a record. Is this somehow already possible today? During a "google" search I came across this article. [https://docs.pushmetrics.io/docs/dataentry-superset/](url) exactly what I'm looking for. unfortunately this doesn't seem to be for Apache Superset :-(

here a GIF from the page https://docs.pushmetrics.io/
![dataentry-superset0](https://user-images.githubusercontent.com/101565081/158167401-4a8c207b-1029-462f-ba84-0ecf4fa598e8.gif)

**Comment by CDT** [comment]
Really need this!

**Comment by mike-fischer1** [comment]
would e helpful

**Comment by zsh-754207424** [comment]
really need this

**Comment by roshanlabh** [comment]
We are also looking for the same feature. Is there any work around of it to achieve the same in superset?

**Reply by BigBatz** [reply]
Hello, I have not pursued this topic any further since then.

**Comment by JustAzes** [comment]
This feature would make the system more flexible. For example, it could be used to correct wrong data entries in QA environments. It would also allow non-technical users, who are familiar with Excel, to interact with the data directly. This can make some processes faster and more user-friendly and empower collaboration and customization. Users could fix mistakes directly without the need for other tools. 

Even though i would like to see this feature, i guess implementing it safely might be challenging. It would require new user roles and permissions. Also the data sources might not (want to) support sending back changes and the changes should be tracked.

**Comment by enescabri** [comment]
This feature wow, it would be great and game changing.

**Comment by CoderSxy** [comment]
I have created a new viz plugin using handsontable to achieve this , but this is a new viz type of chart

**Reply by enescabri** [reply]
Would you mind sharing some of it with us?

**Reply by CoderSxy** [reply]
The code is in the company's private network. We have created a new dashboard type to display handsontable, and added a route and page for the creation of database tables and the configuration of handsontable

**Reply by enescabri** [reply]
Thank you for tip :)
